src/placeos-rest-api/controllers/users.cr

```diff
@@ -225,6 +226,14 @@ module PlaceOS::Api
       render_error(HTTP::Status::BAD_REQUEST, e.message)
     end
 
+    post "/:id/revive", :undestroy do
+      user.deleted = false
+      user.save
+      head :ok
```

Comment:
```suggestion
      save_and_respond(user)
```

Would be it worth responding the model?

The status of the save on line 231, so this could give `200` if it's not working.